lib/momento/cache_client.rb

```diff
@@ -339,9 +341,13 @@ def sorted_set_fetch_by_score(cache_name, sorted_set_name, min_score: nil, max_s
     private
 
     def cache_stub
-      @cache_stub ||= CACHE_CLIENT_STUB_CLASS.new(@cache_endpoint, combined_credentials,
-        timeout: @configuration.transport_strategy.grpc_configuration.deadline
-      )
+      @cache_stubs ||= (1..@num_cache_stubs).map { CACHE_CLIENT_STUB_CLASS.new(@cache_endpoint, combined_credentials,
+        timeout: @configuration.transport_strategy.grpc_configuration.deadline,
+        channel_args: { 'grpc.use_local_subchannel_pool' => 1}
+      ) }
+      @next_cache_stub_index = (@next_cache_stub_index + 1) % @num_cache_stubs
+      puts "Next cache stub index: #{@next_cache_stub_index}"
```

Comment:
Is this print statement supposed to be here?

Reply:
nope thanks